html/webappapis/scripting/events/event-handler-handleEvent-ignored.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>"handleEvent" property of EventHandler should be ignored</title>
+<link rel="help" href="https://html.spec.whatwg.org/#eventhandler">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+"use strict";
+
+test(t => {
+  const el = document.createElement("div");
+  el.onmouseenter = {
+    get handleEvent() {
+      t.step(() => {
```

Comment:
You can use t.unreached_func here as well, no?

Reply:
I guess `t.unreached_func` is unusable with `get` syntax.

Reply:
But why not use the same syntax as below? Is there a difference I'm not understanding?

Reply:
My bad, no difference at all, fixed in ee93d80.

I just find `get` syntax easier (no prototypes and descriptors), but we are using `Object.create` in this test anyway.